environments/contrail/install_vrouter_kmod.yaml

```diff
@@ -397,11 +398,11 @@ resources:
                   ip route add default via $def_gw
                 fi
               fi
-              if [[ ${sriov_on} == 'true' ]]; then
+              if [[ ${sriov_on} == 'True' ]]; then
                 sed 's/^\(GRUB_CMDLINE_LINUX=".*\)"/\1 $KERNEL_ARGS intel_iommu=on iommu=pt"/g' -i /etc/default/grub ;
```

Comment:
remove  iommu=pt  (it is only for 'vfio-pci'  dpdk driver)

Reply:
add a check around sed like:
if ! grep -q 'intel_iommu=on' /etc/default/grub ; then
   sed -i ....
fi
(to avoid additional patching of grub file)